wyckoff_generation/common/utils.py

```diff
@@ -1,3 +1,23 @@
+"""
+The functions ``setup_imports``, ``_get_project_root``, and ``_import_local_file`` were borrowed and modified from
+https://github.com/facebookresearch/fairchem/blob/main/src/fairchem/core/common/utils.py, which is under an MIT License. See below
+
```

Comment:
```suggestion
"""
Parts of this file, the functions ``setup_imports``, ``_get_project_root``, and ``_import_local_file`` , are based on code made available under the MIT license by the fairchem repository by FAIR Chemistry available from their repository as https://github.com/facebookresearch/fairchem/blob/7096ef75aaa0d54b65b223123b72c4d35b2fb5ec/src/fairchem/core/common/utils.py

Modifications Copyright (c) 2025 The High-Throughput Toolkit

The original copyright header from the fairchem file follows below:
```

Reply:
Detta är en tidigare commit
```suggestion
"""
Parts of this file, the functions ``setup_imports``, ``_get_project_root``, and ``_import_local_file`` , are based on code made available under the MIT license by FAIR Chemistry available from their repository as https://github.com/facebookresearch/fairchem/blob/7096ef75aaa0d54b65b223123b72c4d35b2fb5ec/src/fairchem/core/common/utils.py

Modifications Copyright (c) 2025 The High-Throughput Toolkit

The original copyright header from the fairchem file follows below:
```